main.js

```diff
@@ -0,0 +1,11 @@
+const navBar = document.querySelector('#desktop-nav');
+const hamburguerButton = document.querySelector('.menu-bars');
+const hamburguerIcon = document.querySelector('.navbar-icon');
+
+function doSomething() {
```

Comment:
- [ ] I think it would be a better practice if you use more descriptive names for your functions. It will help to keep your code readable and maintainable. Remember that most of the time you will be working with a team.
- [ ] [OPTIONAL] I would recommend using arrow functions instead of regular functions. Not only because of the syntactic sugar, but these are easier to read.